canonical/lxd-ui
# fix(component) dynamic text area will now contain content exactly

## Done

Previously I was setting the text area height to be its scroll height + 1px. This was done so that the scrollbar doesn't show for the text area (if height of container === scroll height of container then the scrollbar shows), but I see this is causing the text area to grow faster than its content. Instead of doing that, we can simply just hide the scrollbar using css.

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Go anywhere that you can find a text area element, copy a body of text and paste into the text area. The text area should grow as tall as the content within.

## Screenshots
[Screencast from 08-12-2023 16:45:42.webm](https://github.com/canonical/lxd-ui/assets/151511568/217b8a35-031a-4bdc-8c00-fb237ba8a74a)
